std_msgs/Header Header
    uint32 seq
    time stamp
    string frame_id

sensor_msgs/PointCloud2[] clusters